src/common/head.tsx

```diff
@@ -161,14 +161,19 @@ export function defaultHeadConfig(
       ...addressTags,
       ...(additionalTags || []),
     ],
-    other: [yaScript(), SchemaBuilder(data)].join("\n"),
+    other: [yaScript(), SchemaBuilder(data), yextensionData(data)].join("\n"),
   };
 }
 
 function yaScript(): string {
   return `<script>window.yextAnalyticsEnabled=false;window.enableYextAnalytics=()=>{window.yextAnalyticsEnabled=true}</script>`;
 }
 
+// Script for passing entity ID to yextension
+function yextensionData(data: TemplateRenderProps): string {
+  return `<script id="yextension-data" data-entity-id="${data.document.uid}"></script>`;
```

Comment:
If this isn't how we're already doing it on live sites, could we name this something more generic to reduce the chances of clients asking us to remove it from their pages? Something like `yext-entity-data` instead of `yextension-data`?